test/e2e/runner/load.go

```diff
@@ -95,32 +96,56 @@ func loadGenerate(ctx context.Context, chTx chan<- types.Tx, multiplier int, siz
 func loadProcess(ctx context.Context, testnet *e2e.Testnet, chTx <-chan types.Tx, chSuccess chan<- types.Tx) {
 	// Each worker gets its own client to each node, which allows for some
 	// concurrency while still bounding it.
-	clients := map[string]*rpchttp.HTTP{}
+
+	clients := make([]*rpchttp.HTTP, 0, len(testnet.Nodes))
+
+	for idx := range testnet.Nodes {
+		if testnet.Nodes[idx].Mode == e2e.ModeSeed {
+			continue
+		}
+		client, err := testnet.Nodes[idx].Client()
+		if err != nil {
+			continue
```

Comment:
Is this worth logging?

Reply:
we never did before? I'm inclined to not change the error conditions where this function logs/panics/errors. Also because this is happening in a loop, I wouldn't want to log a lot of errors in situations where the test would continue to be valid.